docs/next.config.js

```diff
@@ -65,12 +68,6 @@ module.exports = {
     ) {
       const [nextExternals, ...externals] = config.externals;
 
-      if (externals.length > 0) {
```

Comment:
Could we update next to at least 12.1.6? This way the problem with externals would be solved.

Reply:
12.1.6 has problems with CSS generation, see https://github.com/mui/material-ui/issues/33126, but I have just checked and saw that 12.2.0 was released. I will update to that version and see if we still need this.